diff -ur ffmpeg-3.4.2.orig/libswscale/arm/rgb2yuv_neon_16.S ffmpeg-3.4.2/libswscale/arm/rgb2yuv_neon_16.S
--- libswscale/arm/rgb2yuv_neon_16.S	2018-06-29 07:19:34.756576918 -0400
+++ libswscale/arm/rgb2yuv_neon_16.S	2018-06-29 07:20:15.084040570 -0400
@@ -18,6 +18,8 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */

+#include "config.h"
+#if HAVE_AS_DN_DIRECTIVE
 #include "rgb2yuv_neon_common.S"

 /* downsampled R16G16B16 x8 */
@@ -78,3 +80,4 @@
 .endm

     loop_420sp  rgbx, nv12, init, kernel_420_16x2, 16
+#endif
diff -ur ffmpeg-3.4.2.orig/libswscale/arm/rgb2yuv_neon_32.S ffmpeg-3.4.2/libswscale/arm/rgb2yuv_neon_32.S
--- libswscale/arm/rgb2yuv_neon_32.S	2018-06-29 07:19:34.756576918 -0400
+++ libswscale/arm/rgb2yuv_neon_32.S	2018-06-29 07:20:48.227598329 -0400
@@ -18,6 +18,8 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */

+#include "config.h"
+#if HAVE_AS_DN_DIRECTIVE
 #include "rgb2yuv_neon_common.S"

 /* downsampled R16G16B16 x8 */
@@ -117,3 +119,4 @@


     loop_420sp  rgbx, nv12, init, kernel_420_16x2, 32
+#endif
diff -ur ffmpeg-3.4.2.orig/libswscale/arm/swscale_unscaled.c ffmpeg-3.4.2/libswscale/arm/swscale_unscaled.c
--- libswscale/arm/swscale_unscaled.c	2018-06-29 07:19:34.756576918 -0400
+++ libswscale/arm/swscale_unscaled.c	2018-06-29 07:21:29.371047669 -0400
@@ -23,6 +23,7 @@
 #include "libswscale/swscale_internal.h"
 #include "libavutil/arm/cpu.h"

+#if HAVE_AS_DN_DIRECTIVE
 extern void rgbx_to_nv12_neon_32(const uint8_t *src, uint8_t *y, uint8_t *chroma,
                 int width, int height,
                 int y_stride, int c_stride, int src_stride,
@@ -178,3 +179,8 @@
     if (have_neon(cpu_flags))
         get_unscaled_swscale_neon(c);
 }
+#else
+void ff_get_unscaled_swscale_arm(SwsContext* c)
+{
+}
+#endif
