diff --git a/code/Common/ZipArchiveIOSystem.cpp b/code/Common/ZipArchiveIOSystem.cpp
index 7c37a05f..45b877f0 100644
--- a/code/Common/ZipArchiveIOSystem.cpp
+++ b/code/Common/ZipArchiveIOSystem.cpp
@@ -155,7 +155,7 @@ namespace Assimp {
         mapping.ztell_file = (tell_file_func)tell;
         mapping.zseek_file = (seek_file_func)seek;
         mapping.zclose_file = (close_file_func)close;
-        mapping.zerror_file = (error_file_func)testerror;
+        mapping.zerror_file = (testerror_file_func)testerror;
 #else
         mapping.zopen_file = open;
         mapping.zread_file = read;
